### ISSUE TITLE
Use pretrained network

### ISSUE THREAD

**Issue by TheShip1948** [TheShip1948/basicShapesDataset#3]
Use pretrained network to help in basic shapes classification problem

**Comment by TheShip1948** [comment]
Article that I am following: https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html

**Comment by TheShip1948** [comment]
Experiment: 
-----------------
I used one of the techniques posted in keras blog post, but I made a simplification to it. 
Original solution was to mix between augmented image and pre-trained VGG16  architecture. 

Challenge: 
----------------
Generating augmented images on the fly, then use them as input to the network took extremely large time. I don't know the reason yet. 

Solution: 
--------------
I simplified the solution by using my small dataset directly without any data augmentation. The results are impressive, if I just used around 5 epochs after that an extremely weird behavior takes place. The accuracy drops to around 11.49% as if I am using untrained network.

Action Items: 
------------------- 
* Investigate the reason of the degradation of results on using more epochs. 
* Work on enhancing the results (I am eager to use combination of models one of them consists of  one category vs Others)
